addon/store.js

```diff
@@ -61,6 +61,15 @@ var Store = ServiceType.extend({
       });
       this.set("array", {});
     },
+    willDestroy() {
+      this.setProperties({
+        "array": null,
+        "identityMap": null,
+        // Would rather set to null but updateFilters runs after destroy somehow
```

Comment:
@toranb seems that in the test the `updateFilters` method is called after `willDestroy` so needed a couple props to be reset instead of `null`-ed.